### PR TITLE
chromium-chronicle-3

### PR DESCRIPTION
**Pull request by enegiar**
Creating a new branch for the June edition of the Chromium Chronicle

**Target Live Date:** 2019-06-25

### CODE REVIEW

File: src/content/en/updates/2019/06/chromium-chronicle-3 [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+*by Yuke, Roberto and Sajjad in Mountain View*
+
+Tests are critical because they find bugs and regressions, enforce better
+designs and make code easier to maintain. Code coverage helps you ensure
+your tests are thorough.
+
+Chromium CLs can show a line-by-line breakdown of test coverage. You can
+use the code coverage trybot to ensure you only submit well-tested code.
+
+**Insert Image**
```
Comment (enegiar) [inline src/content/en/updates/2019/06/chromium-chronicle-3]: Not sure how to insert an image. Help here would be much appreciated.

File: src/content/en/updates/2019/06/chromium-chronicle-3 [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+To see code coverage for a Chromium CL, trigger the code coverage trybot
+linux-coverage-rel:
+
+Once the build finishes and code coverage data is processed successfully,
+look at the right column of the side by side diff view to see coverage
+information:
+
+**Insert Image**
```
Comment (enegiar) [inline src/content/en/updates/2019/06/chromium-chronicle-3]: Same as above

File: src/content/en/updates/2019/06/chromium-chronicle-3 (hunk 1 above)
Comment (petele) [reply src/content/en/updates/2019/06/chromium-chronicle-3]: To do two side by side images akin what's in the doc

```
<img src="/web/updates/images/2019/06/LEFT_FILENAME.ext" class="attempt-left">
<img src="/web/updates/images/2019/06/RIGHT_FILENAME.ext" class="attempt-right">
<div class="clearfix"></div>
```

Then, drop the image files in `src/content/en/updates/images/2019/06/`, and add them to the PR.

File: src/content/en/updates/2019/06/chromium-chronicle-3 (hunk 2 above)
Comment (petele) [reply src/content/en/updates/2019/06/chromium-chronicle-3]: ``

And similarly, drop it in `src/content/en/updates/images/2019/06/`

File: src/content/en/updates/2019/06/chromium-chronicle-3 [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+description: The Chromium Chronicle, a monthly series geared specifically to Chromium developers — the developers who build the browser.
+
+{# wf_updated_on: 2019-06-18 #}
+{# wf_published_on: 2019-06-25 #}
+{# wf_tags: chromium-chronicle #}
+{# wf_featured_image: /web/updates/images/generic/cr-chron.jpg #}
+{# wf_featured_snippet: Tests are critical because they find bugs and regressions, enforce better designs and make code easier to maintain. This month, we take a look at how to conduct thorough tests with Gerrit #}
+{# wf_blink_components: N/A #}
+
+<style>
```
Comment (petele) [inline src/content/en/updates/2019/06/chromium-chronicle-3]: You can remove this style block since we're not showing any code in this article

Comment (enegiar) [reply src/content/en/updates/2019/06/chromium-chronicle-3]: Done!

File: src/content/en/updates/2019/06/chromium-chronicle-3 (hunk 1 above)
Comment (enegiar) [reply src/content/en/updates/2019/06/chromium-chronicle-3]: Done! Please let me know if I did this correctly.

File: src/content/en/updates/2019/06/chromium-chronicle-3 (hunk 2 above)
Comment (enegiar) [reply src/content/en/updates/2019/06/chromium-chronicle-3]: Done! Please let me know if I did this correctly.

File: src/content/en/updates/2019/06/chromium-chronicle-3.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+*by Yuke, Roberto and Sajjad in Mountain View*
+
+Tests are critical because they find bugs and regressions, enforce better
+designs and make code easier to maintain. Code coverage helps you ensure
+your tests are thorough.
+
+Chromium CLs can show a line-by-line breakdown of test coverage. You can
+use the code coverage trybot to ensure you only submit well-tested code.
+
+<img src="/web/updates/images/2019/06/LEFT_TRYJOB.png.ext" class="attempt-left">
```
Comment (petele) [inline src/content/en/updates/2019/06/chromium-chronicle-3.md]: Remove the `.ext` from the end of the filename.

File: src/content/en/updates/2019/06/chromium-chronicle-3.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+<img src="/web/updates/images/2019/06/RIGHT_TRYJOB.png.ext" class="attempt-right">
```
Comment (petele) [inline src/content/en/updates/2019/06/chromium-chronicle-3.md]: Remove the `.ext` from the end of the filename.

File: src/content/en/updates/2019/06/chromium-chronicle-3.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+<div class="clearfix"></div>
+
+To see code coverage for a Chromium CL, trigger the code coverage trybot
+linux-coverage-rel:
+
+Once the build finishes and code coverage data is processed successfully,
+look at the right column of the side by side diff view to see coverage
+information:
+
+<img src="/web/updates/images/2019/06/CODE_DIFF.png.ext">
```
Comment (petele) [inline src/content/en/updates/2019/06/chromium-chronicle-3.md]: Remove the `.ext` from the end of the filename.

File: src/content/en/updates/2019/06/chromium-chronicle-3.md (hunk 5 above)
Comment (enegiar) [reply src/content/en/updates/2019/06/chromium-chronicle-3.md]: Done!

File: src/content/en/updates/2019/06/chromium-chronicle-3.md (hunk 6 above)
Comment (enegiar) [reply src/content/en/updates/2019/06/chromium-chronicle-3.md]: Done!

File: src/content/en/updates/2019/06/chromium-chronicle-3.md (hunk 4 above)
Comment (enegiar) [reply src/content/en/updates/2019/06/chromium-chronicle-3.md]: Done!

**Comment by enegiar** [comment]
> Can you run the images through [Squoosh](https://squoosh.app/) to compress (and potentially resize) them. JPG seems to be the best format for them, and I'd suggest a max width about 1600px for full width images, and 800px for the side-by-side images.

Done! Added the new images to src/content/en/updates/images/2019/06

**Pull request merged**